addons/website/static/src/core/interaction.js

```diff
@@ -1,11 +1,49 @@
 /**
- * This is the base class to describe interactions. It contains a few helper
- * to accomplish common tasks, such as adding dom listener or waiting for
- * some task to complete
+ * This is the base class to describe interactions. The Interaction class
+ * provides a good integration with the web framework (env/services), a well
+ * specified lifecycle, some dynamic content, and a few helper functions
+ * designed to accomplish common tasks, such as adding dom listener or waiting for
+ * some task to complete.
+ * 
+ * Note that even though interactions are not destroyed in the standard workflow
+ * (a user visiting the website), there are still some cases where it happens:
```

Comment:
We might encounter counter examples with sub-interactions.
There could be a tabbed pane which only runs the interactions of the active tab for example.

Reply:
true. i guess we will rework the comments sometimes